cmd/webserver.go

```diff
@@ -53,6 +54,7 @@ var inst instances.Instances
 var comps components.Components
 var configs configurations.Configurations
 var stats status.Status
+var actor actors.Actors
```

Comment:
I think you can get rid of separate Actors and also Status package. Just expose methods on the existing kube client - that is Instances( Kubernetes )